builtin/aws/ecs/releaser.go

```diff
@@ -51,43 +54,119 @@ func (r *Releaser) Release(
 	}
 	elbsrv := elbv2.New(sess)
 
-	dlb, err := elbsrv.DescribeLoadBalancers(&elbv2.DescribeLoadBalancersInput{
-		LoadBalancerArns: []*string{&target.LoadBalancerArn},
-	})
```

Comment:
Here's the bug. If you specify your own target group arn, `target.LoadBalancerArn` would be an empty string, and you'd get a 400 on this call.